tests/test_media_urlencoded.py

```diff
@@ -37,7 +38,7 @@ def test_urlencoded_form_handler_serialize(data, expected):
 class MediaMirror:
 
     def on_post(self, req, resp):
-        resp.media = req.media
```

Comment:
I haven't checked, but have we left enough places where `media` is referenced as a property?
If not, we should IMHO, since that behaviour is still supported in parallel to `get_media()`.

Reply:
Yes, there are some tests that are specific to this in test_request_media